fix typo (#20831)
mdn/content: files/en-us/web/guide/ajax/getting_started/index.md

const httpRequest = new XMLHttpRequest();
```

> After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:

Reviewer: ```suggestion
After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object to the function called when the request changes state, like this:
```

```js

> httpRequest.onreadystatechange = () => {
>   // Process the server response here.
> };

Reviewer: ```suggestion
function handler() {
  // Process the server response here.
}

httpRequest.onreadystatechange = handler;
```

const httpRequest = new XMLHttpRequest();
```

> After making a request, you will receive a response back. At this stage, you need to tell the XMLHttpRequest object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:

Reviewer: ```suggestion
After making a request, you will receive a response back. At this stage, you need to tell the `XMLHttpRequest` object which JavaScript function will handle the response, by setting the `onreadystatechange` property of the object and naming it after the function to call when the request changes state, like this:
```

```js

> function nameOfTheFunction() {
>     // Process the server response here.
> }
> httpRequest.onreadystatechange = nameOfTheFunction;

Reviewer: If we provide the actual function stub, the `nameOfTheFunction` placeholder doesn't make sense anymore.

```suggestion
httpRequest.onreadystatechange = () => {
  // Process the server response here.
};
```
